src/CMakeLists.txt

```diff
@@ -540,7 +559,7 @@ find_package(Pugixml 1.2 REQUIRED)
 include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
-if(NOT SOURCE_PACKAGE AND NOT APPLE)
+if(NOT SOURCE_PACKAGE AND NOT APPLE AND NOT WIN32)
```

Comment:
> Shadow warnings, like warning: declaration of 'data' shadows a previous local [-Wshadow] const uchar8 *data = mFile->getData(key_off, 1);

They are in rawspeed, and for rawspeed, i keep errors for -Wshadow disabled.
If they are errors for you, something is wrong.